python/lsst/ts/m2/csc.py

```diff
@@ -624,6 +640,16 @@ async def do_disable(self, data: salobj.BaseMsgType) -> None:
 
         await super().do_disable(data)
 
+    def _is_bump_test_done(self) -> bool:
+        """The bump test is done or not.
+
+        Returns
+        -------
+        `bool`
+            True if the bump test is done. Otherwise, False.
+        """
+        return self._task_bump_test.done() or self._task_bump_test.cancelled()
```

Comment:
A "cancelled" task is also considered "done", so you can do just `return self._task_bump_test.done()`

Reply:
Done in the commit: db3ca38. Thanks!